devicon.json

```diff
@@ -3410,6 +3410,27 @@
         "color": "#56c5a8",
         "aliases": []
     },
+    {
+        "name": "ktor",
```

Comment:
The icon object needs to be inserted in the correct place, so that the file is alphabetical

Reply:
Moved :)